setcontext/main.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+import sys
+import pathlib
+
+#Add pytest
+#Build framework for  commandline tools.
+
+"""
+function to handle bad version string
+"""
+def validate_version_str(version_string: str):
+    #replace with regex
+    if not version_string.startswith("v"):
+        sys.exit(f"version string: {version_string} doesn't start with v")
+    if not version_string[1:].isdigit():
+        sys.exit(f"version string: {version_string} doesnt end in numerics 00 - 99")
+    if not len(version_string) == 4:
+        sys.exit(f"version string: {version_string} doesnt end in numerics 000 - 999")
+
+if __name__ == '__main__':
+
+    project = None
+    service = None
+    version = None
+
+    #replace with argparse for better error handling and help?
```

Comment:
I know it is a little early in development but it is a good practice to document what is the expected format for your input. Will help readers know exactly what to pass. 

```
# Accepted formats
# <project>:<service>:<version>
# <project>:<service>
# <project>

```